lib/hanami/routes.rb

```diff
@@ -38,7 +40,7 @@ def self.define(&block)
 
     # @api private
     def self.routes
-      @_routes || raise(<<~MSG)
+      (@_routes || build_routes) || raise(<<~MSG)
```

Comment:
Should we also remove the `.define` method (above) as well as this `@_routes` ivar?

Unless there's potential drawbacks to the new class-level approach of routes definition, it feels like we'd be smart to provide just one way to define routes in this class.